### ISSUE TITLE
Add Man-Page for Unix-Like Systems

### ISSUE THREAD

**Issue by Breakthrough** [Breakthrough/PySceneDetect#9]
As per title, add a man-page synopsis so users can call `man scenedetect` for detailed usage help.

Perhaps use Pandoc to convert a designated Markdown file during package creation:
http://www.pandoc.org/demos.html

**Comment by SurenSN** [comment]
Hi,

I am trying to write the man page(If i may).

I have a few questions, which are as follows:

1. Should the man page be a separate file?
    If yes, maintaining multiple documents (manpage.md, readme.md and usage.md)
    with some common information would be tedious.

2. Can the information required to form manpage.md be extracted from existing
    documents/code?

This is my first effort to make an OSS contribution. 
I am not sure if this is a good first issue to pick up.
Please guide.

**Comment by Breakthrough** [comment]
Absolutely, feel free!  (Make sure to [include all the CLI parameters](http://pyscenedetect.readthedocs.io/en/latest/reference/command-line-params/) if you do so, can also get it from v0.5-beta-1 by calling `help all`).

In response to your questions:

  1. No *if possible*, but I'm not sure if that is... One option is to call `scenedetect help all` and use that as the `man scenedetect` text, which I am more than open to.  If this needs to be saved as an actual manpage file and updated as part of the release process, that's okay with me so long as it can be automated.

  2. Absolutely, yes!

Lastly, please feel free to take on the issue.  Have confidence 👍
